com.unity.shadergraph/Editor/Drawing/Controllers/BlackboardController.cs

```diff
@@ -192,13 +372,36 @@ internal SGBlackboard blackboard
             get => m_Blackboard;
             private set => m_Blackboard = value;
         }
+        public string GetFirstSelectedCategoryGuid()
```

Comment:
This looks like it returns `String.Empty` if multiple categories are selected. Why is this the case?